frame/elections-phragmen/src/weights.rs

```diff
@@ -228,3 +228,86 @@ impl WeightInfo for () {
 			.saturating_add(RocksDbWeight::get().writes((1 as Weight).saturating_mul(c as Weight)))
 	}
 }
+
+pub struct MockWeightInfo<T>(PhantomData<T>);
+impl<T: frame_system::Config> WeightInfo for MockWeightInfo<T> {
+	fn vote_equal(v: u32, ) -> Weight {
+		(45_157_000 as Weight)
+			// Standard Error: 6_000
+			.saturating_add((399_000 as Weight).saturating_mul(v as Weight))
+			.saturating_add(RocksDbWeight::get().reads(5 as Weight))
+			.saturating_add(RocksDbWeight::get().writes(2 as Weight))
```

Comment:
```suggestion
		(45_157_000 as Weight)
			unreachable!();
```

Same should be applied to all the ones that are not used.